src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/ContentDbContextModelSnapshot.cs

```diff
@@ -4006,9 +4004,15 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             modelBuilder.Entity("GovUk.Education.ExploreEducationStatistics.Content.Model.Comment", b =>
                 {
-                    b.HasOne("GovUk.Education.ExploreEducationStatistics.Content.Model.IContentBlock", null)
+                    b.HasOne("GovUk.Education.ExploreEducationStatistics.Content.Model.IContentBlock", "ContentBlock")
                         .WithMany("Comments")
-                        .HasForeignKey("IContentBlockId");
+                        .HasForeignKey("ContentBlockId")
+                        .OnDelete(DeleteBehavior.Cascade)
```

Comment:
Would this mean that deleting this comment would then in turn delete the associated ContenBlock?

Reply:
It's the other way around, deleting the foreign key ContentBlock will delete all the linked Comments. It could be set, for example, to SetNull in which case deleting the ContentBlock would set the IContentBlockId on each linked Comment to null.